Sources/TensorFlow/Operators/Basic.swift

```diff
@@ -1325,34 +1325,37 @@ extension Tensor.IndexPath {
 //===------------------------------------------------------------------------------------------===//
 
 extension Tensor {
-  /// Returns `true` if the given axis is in the range `[-rank, rank)`.
+  /// Returns `true` if `axis` is in `-rank..<rank`, indicating it refers to an axis of `self`.
```

Comment:
IMO exposing the implementation in the summary has negative value; it merely obscures the high-level semantics and breaks the abstraction provided by the function.  When I'm reading code that uses it, I want to know what calling this method _means,_ not how it does its work.  I can understand needing to have the information *somewhere*, but I'd expect to find the information about valid axis in documentation on `Tensor`, which surely has `public` APIs that depend on understanding thisrelationship.  If this *were* really the only place to describe it, I'd get it out of the summary and put it in a separate "paragraph."

Reply:
Very much improved, thanks!  I left some notes in case you care.